registry/src/skills.json

```
@@ -7,6 +7,92 @@
         "author": null,
         "description": "Enables your agent to fetch token details for any token on Base",
         "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details",
+        "imageUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details/images/logo.png"
+    },
+    {
+        "pluginId": "ba2e0746-d089-4539-a23b-84882bc8eafd",
+        "name": "@moxie-protocol/plugin-betswirl",
+        "version": "0.0.1",
+        "displayName": "Play Games of Luck",
```

Recommend to just name this as `BetSwirl` to fit the description and everything else, I think people might be familiar with this brand name as well

```suggestion
        "displayName": "BetSwirl",
```